bindinfo/handle.go

```diff
@@ -566,7 +566,11 @@ func copyBindRecordUpdateMap(oldMap map[string]*bindRecordUpdate) map[string]*bi
 func (c cache) getBindRecord(hash, normdOrigSQL, db string) *BindRecord {
 	bindRecords := c[hash]
 	for _, bindRecord := range bindRecords {
+<<<<<<< HEAD
```

Comment:
Do we need to pick it into v5.0-rc? If it is, please solve these conflicts ~ @rebelice